cluster/addons/volumesnapshots/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
is there a way to avoid copying crds here but rather referring to a specific snapshot version to avoid inconsistency?

Reply:
This is actually not exactly same as the CRDs in external-snapshotter repo.  This needs a special label to be picked up at install time and I also have to remove timestamp field.

Reply:
Yes, see my comment about updating the RBAC file in this PR... personally I would prefer an "upstream" first approach: always make changes in the individual repo first (because that is the canonical definition of the RBAC files!), then copy into k/k.

With the CRDs it's going to be worse if they aren't even 1:1 copies from upstream. All I can think of is making changes upstream, overwrite the file in k/k, then use "git add -i" to commit only those changes that were made upstream since the last import.